share/qt/Info.plist.in

```diff
@@ -29,10 +29,10 @@
   <string>????</string>
 
   <key>CFBundleExecutable</key>
-  <string>PAIcoin-Core</string>
```

Comment:
Any reason that it can't be `PAI Up` without the dash?

Reply:
Given the Make limitations in using the spaces in names, this can be achieved with a post-processing script.